cmd/minikube/cmd/status.go

```diff
@@ -84,18 +86,18 @@ const (
 	Paused   = 418 // I'm a teapot!
 
 	// 5xx signifies a server-side error (that may be retryable)
-	Error               = 500
-	InsufficientStorage = 507
```

Comment:
InsufficientStorage should stay 507 -- these are HTTP style error codes, not UNIX error codes.

Reply:
Thanks for catching that, fixed